.github/workflows/pr-preview.yml

```diff
@@ -0,0 +1,93 @@
+# .github/workflows/pr-preview.yml
+name: Deploy PR previews
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - closed
+    paths:
+      - '**/*.md'
+
+concurrency: preview-${{ github.ref }}
```

Comment:
Lets do
```
concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: true
```

This will cancel previous action if there is a commit in between